[CSS Zoom] Rename 'enableEvaluationTimeZoom' flag

vitorroriz opened the pull request:
#### a7f9455f5fa5c6ab72e972909771f61cc60ff7d2
<pre>
[CSS Zoom] Rename &apos;enableEvaluationTimeZoom&apos; flag
<a href="https://rdar.apple.com/163031740">rdar://163031740</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=301093">https://bugs.webkit.org/show_bug.cgi?id=301093</a>

Reviewed by Matthieu Dubet and Antoine Quint.

Renaming &apos;enableEvaluationTimeZoom&apos; to `evaluationTimeZoomEnabled`
for a better conformity with WebKit&apos;s convention.

* Source/WebCore/platform/graphics/FontDescription.cpp:
(WebCore::m_evaluationTimeZoom):
(WebCore::m_enableEvaluationTimeZoom): Deleted.
* Source/WebCore/platform/graphics/FontDescription.h:
(WebCore::FontDescription::computedSizeForRangeZoomOption const):
(WebCore::FontDescription::evaluationTimeZoom const):
(WebCore::FontDescription::setEvaluationTimeZoom):
(WebCore::FontDescription::enableEvaluationTimeZoom const): Deleted.
(WebCore::FontDescription::setEnableEvaluationTimeZoom): Deleted.
* Source/WebCore/rendering/style/RenderStyle.cpp:
(WebCore::RenderStyle::setEvaluationTimeZoom):
(WebCore::RenderStyle::setEnableEvaluationTimeZoom): Deleted.
* Source/WebCore/rendering/style/RenderStyle.h:
* Source/WebCore/rendering/style/RenderStyleInlines.h:
(WebCore::RenderStyle::evaluationTimeZoom const):
(WebCore::RenderStyle::usedZoomForLength const):
(WebCore::RenderStyle::enableEvaluationTimeZoom const): Deleted.
* Source/WebCore/rendering/style/StyleRareInheritedData.cpp:
(WebCore::StyleRareInheritedData::StyleRareInheritedData):
(WebCore::StyleRareInheritedData::operator== const):
(WebCore::StyleRareInheritedData::dumpDifferences const):
* Source/WebCore/rendering/style/StyleRareInheritedData.h:
* Source/WebCore/style/StyleResolveForDocument.cpp:
(WebCore::Style::resolveForDocument):
* Source/WebCore/style/values/primitives/StylePrimitiveNumericTypes+Conversions.cpp:
(WebCore::Style::shouldUseEvaluationTimeZoom):

Canonical link: <a href="https://commits.webkit.org/301886@main">https://commits.webkit.org/301886@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7ec28e80c06254d1a8df742e8043cd05dd565a9b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/127362 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/47010 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/38146 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/134426 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/78917 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/6966f087-c471-408f-b08b-25f776a074ed) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/129234 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/47624 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/55533 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/96931 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/64926 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/9c7f1742-4d36-4f70-89b0-319aab1e9dc6) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/130310 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/38109 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/114073 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77425 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/611e8fb3-b87e-48d8-8cec-9db5f215d4d7) 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/36958 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/32188 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/77808 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/107969 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/32589 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/136909 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/54021 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/41628 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105451 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/54532 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/110422 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/105128 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/26805 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/50648 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/29099 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/51600 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/53958 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/60045 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/53191 "Built successfully") | | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/56649 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/54951 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

mdubet commented on Source/WebCore/platform/graphics/FontDescription.h:
```diff
@@ -145,7 +145,7 @@ class FontDescription {
     void setFontPalette(const FontPalette& fontPalette) { m_fontPalette = fontPalette; }
     void setFontSizeAdjust(FontSizeAdjust fontSizeAdjust) { m_sizeAdjust = fontSizeAdjust; }
-    void setEnableEvaluationTimeZoom(bool enableEvaluationTimeZoom) { m_enableEvaluationTimeZoom = enableEvaluationTimeZoom; }
+    void setEvaluationTimeZoom(bool evaluationTimeZoom) { m_evaluationTimeZoom = evaluationTimeZoom; }
```
We sometimes use a `bool = true` to be able to write `foo.setEvaluationTimeZoom()`  which reads nicely

graouts reviewed (changes requested):
I don't think the new name is suggestive of a boolean flag but rather of a numeric zoom value.

I suggest we use `shouldUseEvaluationTimeZoom` as it's pretty idiomatic and is already used in `StylePrimitiveNumericTypes+Conversions.cpp‎`.

nt1m commented on Source/WebCore/style/values/backgrounds/StyleLineWidth.cpp:
```diff
@@ -57,7 +57,7 @@ auto CSSValueConversion<LineWidth>::operator()(BuilderState& state, const CSSVal
     }
 
-    if (shouldUseEvaluationTimeZoom(state))
+    if (shouldUseEvaluationTimeZoomEnabled(state))
```
`shouldUseEvaluationTimeZoomEnabled` 

-> `evaluationTimeZoomEnabled` or `shouldUseEvaluationTimeZoom` ?

Both seems weird.

nt1m commented on Source/WebCore/style/values/inline/StyleLineHeight.cpp:
```diff
@@ -78,7 +78,7 @@ auto CSSValueConversion<LineHeight>::operator()(BuilderState& state, const CSSPr
     if (primitiveValue.isPercentage()) {
         // FIXME: percentage should not be restricted to an integer here.
-        auto textZoom = shouldUseEvaluationTimeZoom(state) ? conversionData.zoom() : 1.0f;
+        auto textZoom = shouldUseEvaluationTimeZoomEnabled(state) ? conversionData.zoom() : 1.0f;
```
Can you change this as well?

vitorroriz committed:
[CSS Zoom] Rename 'enableEvaluationTimeZoom' flag
rdar://163031740
https://bugs.webkit.org/show_bug.cgi?id=301093

Reviewed by Matthieu Dubet and Antoine Quint.

Renaming 'enableEvaluationTimeZoom' to `evaluationTimeZoomEnabled`
for a better conformity with WebKit's convention.

* Source/WebCore/platform/graphics/FontDescription.cpp:
(WebCore::m_evaluationTimeZoom):
(WebCore::m_enableEvaluationTimeZoom): Deleted.
* Source/WebCore/platform/graphics/FontDescription.h:
(WebCore::FontDescription::computedSizeForRangeZoomOption const):
(WebCore::FontDescription::evaluationTimeZoom const):
(WebCore::FontDescription::setEvaluationTimeZoom):
(WebCore::FontDescription::enableEvaluationTimeZoom const): Deleted.
(WebCore::FontDescription::setEnableEvaluationTimeZoom): Deleted.
* Source/WebCore/rendering/style/RenderStyle.cpp:
(WebCore::RenderStyle::setEvaluationTimeZoom):
(WebCore::RenderStyle::setEnableEvaluationTimeZoom): Deleted.
* Source/WebCore/rendering/style/RenderStyle.h:
* Source/WebCore/rendering/style/RenderStyleInlines.h:
(WebCore::RenderStyle::evaluationTimeZoom const):
(WebCore::RenderStyle::usedZoomForLength const):
(WebCore::RenderStyle::enableEvaluationTimeZoom const): Deleted.
* Source/WebCore/rendering/style/StyleRareInheritedData.cpp:
(WebCore::StyleRareInheritedData::StyleRareInheritedData):
(WebCore::StyleRareInheritedData::operator== const):
(WebCore::StyleRareInheritedData::dumpDifferences const):
* Source/WebCore/rendering/style/StyleRareInheritedData.h:
* Source/WebCore/style/StyleResolveForDocument.cpp:
(WebCore::Style::resolveForDocument):
* Source/WebCore/style/values/primitives/StylePrimitiveNumericTypes+Conversions.cpp:
(WebCore::Style::shouldUseEvaluationTimeZoom):

Canonical link: https://commits.webkit.org/301886@main
